src/blocks.zig

```diff
@@ -114,6 +114,9 @@ var _modeData: [maxBlockCount]u16 = undefined;
 var _lodReplacement: [maxBlockCount]u16 = undefined;
 var _opaqueVariant: [maxBlockCount]u16 = undefined;
 var _friction: [maxBlockCount]f32 = undefined;
+
+var _inventorySize: [maxBlockCount]?u32 = undefined;
```

Comment:
This is configuration data specific to the chest entity data class. It should be stored in the `EntityDataClass` using a `*anyopaque`, and we need a `loadFromZon` in the VTable method that loads this data.